### DISCUSSION TITLE
Timezone database is corrupt

### DISCUSSION THREAD

**Discussion by yuanRT** [Novik/ruTorrent#2423] [Q&A] [answered]
### Please complete the following tasks.

- [X] Web browser cache cleared
- [X] Link provided to install script if applicable
- [X] Web browser, ruTorrent, PHP and OS version provided

### Tell us about your environment

Web Browser: Edge 108.0.1462.42
ruTorrent: ruTorrent-4.0-beta3
PHP: PHP 8.1.13 (cgi-fcgi) (built: Dec  6 2022 10:45:12)
OS: OpenWrt 22.03.2 r19803-9a599fee93
rtorrent: 0.9.8-2
web server: lighttpd/1.4.67 (ssl)

### Tell us how you installed ruTorrent

using winscp, copying ruTorrent-4.0-beta3.tar.gz to /www
cd /www
tar -xzf ruTorrent-4.0-beta3.tar.gz
rm ruTorrent-4.0-beta3.tar.gz
mv ruTorrent-4.0-beta3 ruTorrent

### Describe the bug

After all settings is done, start lighttp and php-fpm only, the web server seems ok.
Then start rtorrent, something goes wrong, web browser tell error 500, getting plugins error.
View /var/log/lighttpd/error.log
Find error log as follows:

FastCGI-stderr:#0 /www/ruTorrent/php/settings.php(366): getdate()
FastCGI-stderr:#1 /www/ruTorrent/plugins/rss/rss.php(887): rTorrentSettings->getScheduleCommand('rss', 30, 'execute2={sh,-c...', 0)
FastCGI-stderr:#2 /www/ruTorrent/plugins/rss/init.php(11): rRSSManager->setHandlers()
FastCGI-stderr:#3 /www/ruTorrent/php/getplugins.php(495): require_once('/www/ruTorrent/...')
FastCGI-stderr:#4 {main}
FastCGI-stderr:  thrown in /www/ruTorrent/php/settings.php on line 366
FastCGI-stderr:PHP Fatal error:  Uncaught Error: Timezone database is corrupt. Please file a bug report as this should never happen in /www/ruTorrent/php/settings.php:366
FastCGI-stderr:Stack trace:
FastCGI-stderr:#0 /www/ruTorrent/php/settings.php(366): getdate()
FastCGI-stderr:#1 /www/ruTorrent/plugins/trafic/init.php(8): rTorrentSettings->getScheduleCommand('trafic', 15, 'execute2={sh,-c...')
FastCGI-stderr:#2 /www/ruTorrent/php/getplugins.php(495): require_once('/www/ruTorrent/...')
FastCGI-stderr:#3 {main}
FastCGI-stderr:  thrown in /www/ruTorrent/php/settings.php on line 366
FastCGI-stderr:PHP Fatal error:  Uncaught Error: Timezone database is corrupt. Please file a bug report as this should never happen in /www/ruTorrent/php/settings.php:366

Then disable plugin rss and trafic, it get works.

**Comment by koblack** [comment, marked as answer]
try
```
opkg install zoneinfo-all
```

and in
```
/etc/php.ini
```

change accordingly
```
date.timezone = ...
```

**Comment by yuanRT** [comment]
It works! Really appreciate!
